BRIDGE.md

```diff
@@ -6,11 +6,11 @@
 
 Simple plug-n-play server for supporting [Lightning Address](https://lightningaddress.com) protocol.
 
-> At the end of this flow you will be able to receive payments at `any_name@domain.com`.
+> At the end of this flow you will be able to receive payments at `any_name@example.org`.
 
 ## General
 
-A server that allows you to receive payments at `yourname@yourdomain.com` noncustodially. The Bridge Server serves the necessary JSON and then uses RPC calls to connect to your node and fetch invoices on demand.
+A server that allows you to receive payments at `yourname@yourexample.org` noncustodially. The Bridge Server serves the necessary JSON and then uses RPC calls to connect to your node and fetch invoices on demand.
```

Comment:
I think in this specific case, `domain` makes more sense than `example`, but I also prefer `.org`

Reply:
Technically `.org` `.com`, or any TLD would work here. `.com` is just the most widely known so no way to misunderstand what it represents. Probably best to stick to the most known for this.